python_examples/rebound.py

```diff
@@ -127,4 +149,158 @@ def step():
 def integrate(tmax):
     libias15.integrate(c_double(tmax))
 
+TWOPI = 2.*numpy.pi
+def mod2pi(f):
+    while f<0.:
+        f += TWOPI
+    while f>TWOPI:
+        f -= TWOPI
+    return f
+
+def get_E(e,M):
+    E = M if e<0.8 else numpy.pi
+    
+    F = E - e*numpy.sin(M) - M
+    for i in range(100):
+        E = E - F/(1.0-e*numpy.cos(E))
+        F = E - e*numpy.sin(E) - M
+        if numpy.fabs(F)<1e-16:
+            break
+    E = mod2pi(E)
+    return E
+
+def init_planet(star,       # central object (rebound.Particle object)
+                a,          # semimajor axis
+                e=None,     # eccentricity
```

Comment:
If there are no warning messages, one could change this line to 

    e=0.

And get rid of the extra line below.